compiler/qsc_doc_gen/src/generate_docs.rs

```diff
@@ -28,9 +29,38 @@ struct Compilation {
 
 impl Compilation {
     /// Creates a new `Compilation` by compiling sources.
-    pub(crate) fn new() -> Self {
+    pub(crate) fn new(
+        additional_sources: Option<SourceMap>,
```

Comment:
This function was used to generate API documentation for the standard library only. Now it can generate documentation for the standard library and additional sources. But I see how this name can be confusing. I don't want to name them just 'source' because the whole standard library is always included. I added comment to the generate_docs function below in this file.